src/Model/API/BEditaClient.php

```diff
@@ -241,6 +241,37 @@ public function deleteObject($id, $type)
         return $this->delete(sprintf('/%s/%s', $type, $id));
     }
 
+    /**
+     * Remove an object => permanently remove object from trashcan.
+     *
+     * @param int|string $id Object id
+     * @return array Response in array format
+     */
+    public function remove($id)
+    {
+        return $this->delete(sprintf('/trash/%s', $id));
+    }
+
+    /**
+     * Restore object from trash
+     *
+     * @param int|string $id Object id
+     * @param string $type Object type name
+     * @return array Response in array format
+     */
+    public function restoreObject($id, $type = 'objects')
```

Comment:
since `objects` is abstract I would not allow empty `$type` argument
BTW: does restore `/objects/{id}` work? :thinking: